STM32F405RGTx/Core/Src/rtos_threads/bist_thread.cpp

```diff
@@ -143,3 +151,22 @@ void BistThread::_rgb() {
 
 }
 
+void BistThread::_toggle_fc() {
+    uint8_t buff[3];
+    uint32_t len = 3;
```

Comment:
Should we maybe increase the buffer length just since from how the _sinput method is implemented, if you type `off` it will instantly submit. Like make it four should be enough I think

Reply:
maybe like 8-10 ish imo